app/views/TaxCalcBreakdown.scala.html

```diff
@@ -35,7 +35,7 @@
     <tr class="govuk-table__row">
         <td class="govuk-table__cell">
         @messages(
-            if(scottishRegime && band.name == "ART") {
+            if(scottishRegime && (band.name == "ART" || band.name == "AVRT" && taxYear >= 2024)) {
```

Comment:
Would this be more readable like "if (scottishRegime && (band.name == "ART" || (band.name == "AVRT" && taxYear >= 2024)))". Also, can the magic number be removed with a valid name to understand the criteria like MIN_YEAR_FOR_SCOTTISH_REGIME or a better one?